packages/odyssey-react-mui/src/labs/AppTile.tsx

```diff
@@ -169,7 +169,15 @@ const AppTile = ({
   );
 
   return (
-    <MuiCard className="isClickable">
+    <MuiCard
+      className="isClickable"
```

Comment:
@stephanieyoshimoto-okta We can remove the className here. Can just do
```js
"&:hover": {
  "box-shadow": odysseyDesignTokens.DepthMedium,
},
```

Reply:
Or, better yet, we could move all this into a styled component. That is the method we've been leaning towards when we can. Let me know if you have questions about what that looks like and/or have any issues converting this to a styled component